### PR TITLE
doc(okd): add a section to deploy Longhorn by the manifest

### PR DESCRIPTION
**Pull request by mantissahz**
#### Which issue(s) this PR fixes:

Issue # longhorn/longhorn#8300

#### What this PR does / why we need it:

Users can know how to deploy Longhorn with the manifest file.

### CODE REVIEW

File: content/docs/1.7.0/advanced-resources/os-distro-specific/okd-support.md [hunk 1]
```diff
@@ -26,6 +30,24 @@ And then install Longhorn with setting ***openshift.enabled*** true:
   helm install longhorn longhorn/longhorn --namespace longhorn-system --create-namespace --set openshift.enabled=true
 ```
 
+### Install With `oc` Command
+
+Install Longhorn on OKD Kubernetes cluster using this command:
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/os-distro-specific/okd-support.md:35]:
```suggestion
You can install Longhorn on [OKD](https://www.okd.io/) clusters using the following command:
```

File: content/docs/1.7.0/advanced-resources/os-distro-specific/okd-support.md [hunk 2]
```diff
@@ -26,6 +30,24 @@ And then install Longhorn with setting ***openshift.enabled*** true:
...
+
+One way to monitor the progress of the installation is to watch pods being created in the `longhorn-system` namespace:
+
+  ```shell
+    oc get pods \
+    --namespace longhorn-system \
+    --watch
+  ```
+
+See more details at [Install With Kubectl](../../../deploy/install/install-with-kubectl)
```
Comment (jillian-maroket) [inline content/docs/1.7.0/advanced-resources/os-distro-specific/okd-support.md:49]:
```suggestion
For more information, see [Install with Kubectl](../../../deploy/install/install-with-kubectl).
```

**Review by jillian-maroket — COMMENTED** [review]
Just a few nits

File: content/docs/1.7.0/advanced-resources/os-distro-specific/okd-support.md (hunk 1 above)
Comment (mantissahz) [reply content/docs/1.7.0/advanced-resources/os-distro-specific/okd-support.md]: Modified. Thanks.

File: content/docs/1.7.0/advanced-resources/os-distro-specific/okd-support.md (hunk 2 above)
Comment (mantissahz) [reply content/docs/1.7.0/advanced-resources/os-distro-specific/okd-support.md]: Modified. Thanks.

**Review by jillian-maroket — APPROVED** [review]
LGTM. Thanks!

**Review by derekbit — APPROVED** [review]

**Pull request merged**